### PR TITLE
inventory item for file previewers

### PR DESCRIPTION
**Pull request by qqmyers**
## New Contributors

Welcome! New contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md), especially the section on pull requests where we encourage you to reach out to other developers before you start coding. Also, please note that we measure code coverage and prefer you write unit tests. Pull requests can still be reviewed without tests or completion of the checklist outlined below. Thanks!

## Related Issues

- connects to #5738 Add previewers to external tool inventory

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: http://guides.dataverse.org/en/latest/developers/sql-upgrade-scripts.html
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.3.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: doc/sphinx-guides/source/installation/external-tools.rst [hunk 1]
```diff
@@ -17,6 +17,8 @@ Support for external tools is just getting off the ground but the following tool
 
 - `Whole Tale <https://wholetale.org>`_: a platform for the creation of reproducible research packages that allows users to launch containerized interactive analysis environments based on popular tools such as Jupyter and RStudio. Using this integration, Dataverse users can launch Jupyter and RStudio environments to analyze published datasets. For more information, see the `Whole Tale User Guide <https://wholetale.readthedocs.io/en/stable/users_guide/integration.html>`_.
 
+- `File Previewers <https://github.com/QualitativeDataRepository/dataverse-previewers>`_: A set of tools that display the content of files - including audio, html, `Hypothes.is <https://hypothes.is/>` annotations, images, PDF, text, video - allowing them to be viewed without downloading. The previewers can be run directly from github.io, so the only required step is using the Dataverse API to register the ones you want to use. Documentation, including how to optionally brand the previewers, and an invitation to contribute through github are in the README.md file.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/installation/external-tools.rst]: First, the previewers are awesome. Thanks! My only comment is that while it doesn't look like a download, the download count is incremented when you explore. You **could** say something like "all within your browser" if you want but I'm approving this. Thanks!

Comment (qqmyers) [reply doc/sphinx-guides/source/installation/external-tools.rst]: That's odd - the previewers  use ?gbrecs=false when pulling content for the preview and only use ?gbrecs=true if/when you hit the download button. Is that not the right way to do it?

Comment (qqmyers) [reply doc/sphinx-guides/source/installation/external-tools.rst]: Which is exactly backwards - gbrecs=true suppresses counting the download - I've flipped these in the previewers so this now works correctly...

Comment (pdurbin) [reply doc/sphinx-guides/source/installation/external-tools.rst]: @qqmyers oh! Thanks!

**Pull request merged**